nautobot/cloud/models.py

```diff
@@ -41,3 +42,35 @@ def __str__(self):
     @property
     def display(self):
         return f"{self.provider}: {self.name} - {self.account_number}"
+
+
+@extras_features(
+    "custom_links",
+    "custom_validators",
+    "export_templates",
+    "graphql",
+    "webhooks",
+)
+class CloudType(PrimaryModel):
+    name = models.CharField(max_length=CHARFIELD_MAX_LENGTH, help_text="The name of this Cloud Account.", unique=True)
+    description = models.CharField(max_length=CHARFIELD_MAX_LENGTH, blank=True)
+    provider = models.ForeignKey(
+        to="dcim.Manufacturer",
+        on_delete=models.PROTECT,
+        related_name="cloud_types",
+    )
+    extras_config_schema = models.JSONField(null=True, blank=True)
+    content_types = models.ManyToManyField(
+        to=ContentType,
+        help_text="The content type(s) to which this model applies.",
+    )
+
+    class Meta:
+        ordering = ["provider", "name"]
+
+    def __str__(self):
+        return f"{self.provider}: {self.name}"
+
+    @property
+    def display(self):
+        return str(self)
```

Comment:
Similar to my comment on the parent PR - I'd personally prefer `str` to just use `name`, leaving `display` for the more verbose string.